sdk/src/rent_utils.rs

```diff
@@ -0,0 +1,17 @@
+use crate::account::KeyedAccount;
```

Comment:
Can this fn just live in `rent.rs`?

Reply:
you mean `rent_calculator.rs` right? I didn't put it there, because `rent_calculator.rs` as per name should only be dealt with the `RentCalculator` object IMHO. 

One other way to do this, is to rename `rent_calculator.rs` to `rent.rs` and put `RentCalculator` as well as this fn there. Wdyt?

Reply:
I was thinking `rent.rs`, because the inputs to `verify_rent_exemption()` do require `rent.rs`

Reply:
That's `sysvar/rent.rs`.

Reply:
yeah, same file that `from_keyed_account` was added to

Reply:
(basically creating a "util" file is asking for somebody else to come along and dump other random stuff into it, then we end up with a mess.  I'm trying to preempt that)

Reply:
done